imcache_test.go

```diff
@@ -941,6 +942,53 @@ func TestImcache_GetAll_SlidingExpiration(t *testing.T) {
 	}
 }
 
+//GetAndRemove Tests Init
+func TestImcache_GetAndRemove(t *testing.T) {
+	for _, cache := range caches {
+		t.Run(cache.name, func(t *testing.T) {
+			c := cache.create()
+			c.Set("foo", "foo", WithNoExpiration())
+			c.Set("foobar", "foobar", WithExpiration(time.Hour))
+
+			//get foobar item and delete it
```

Comment:
nit: those tests are really straightforward, comments here are not needed, here and below.